docs/site/tutorials/todo/todo-tutorial-putting-it-together.md

```diff
@@ -42,6 +42,10 @@ Server is running at http://127.0.0.1:3000
 Next, you can use the [API Explorer](http://localhost:3000/explorer) to browse
 your API and make requests!
 
+{% include note.html content="
```

Comment:
@raymondfeng , @dhmlau , I felt I needed to add this note because sometimes the default filter results in a /todos/count of zero, and when I clear it, the count is correct. For the GET operations, same thing, and some endpoints, the server doesn't like the filter and gives 500. Are these defaults provided by swagger ui?

Reply:
What default values are you seeing? They are probably set by https://github.com/strongloop/loopback-next/blob/ec386c15bce904c770a9be51f21d4ff3592dd1af/packages/repository-json-schema/src/filter-json-schema.ts. See the `examples` value.

Reply:
GET /todos
has a default filter of:

```
{
  "offset": 0,
  "limit": 100,
  "skip": 0,
  "where": {
    "additionalProp1": {}
  }
}
```

If user presses [Execute] , it returns an empty array.

If user clears the filter, presses [Execute], then the proper number of todos is returned.

Reply:
GET /todos/count   
has a where clause of

```
{
  "additionalProp1": {}
}
```
when user presses [Execute] button, it returns a count of 0
If we clear the where clause, then press [Execute], it returns the proper count.

Reply:
so perhaps my note.html should be:

```
{% include note.html content=" When using the API Explorer, be sure to clear out any default `filter` or `where` object if you do not intend on using them as they may affect your result. " %}


```

Reply:
We can also reword it as clearing out the filters so that you'll see all the data.